### ISSUE TITLE
Can't use auto completion when login to remote server through SSH

### ISSUE THREAD

**Issue by xiaotianhu** [shockone/black-screen#908]
I uses shell mostly on remote server through SSH,when I login to remote server,the black-screen just goes like traditional shell,and no auto completion at all.Is this my fault?Caused by ZSH or something else?

**Comment by drew-gross** [comment]
No, not your fault. This is because SSH itself is a program that is running, and Black Screen can't autocomplete inside other programs. I've been thinking of ways to autocomplete inside SSH, but haven't started implementing anything yet.
